components/brave_wallet/common/web3_provider_constants.h

```diff
@@ -38,6 +38,9 @@ constexpr char kEthGetBlockByNumber[] = "eth_getBlockByNumber";
 constexpr char kEthBlockNumber[] = "eth_blockNumber";
 constexpr char kEthSign[] = "eth_sign";
 constexpr char kPersonalSign[] = "personal_sign";
+constexpr char kEthSignTypedData[] = "eth_signTypedData";
```

Comment:
Per Slack, I don't think MM sets it to v3. So let's just not handle `eth_signTypedData` yet.

Reply:
fixed in 6bbb1c4516cb09859c699807dc35b5fd17d06b54